app/views/api/services/show.html.slim

```diff
@@ -161,7 +161,6 @@ section class="service-widget"
           - [:buyers_manage_keys, :buyers_manage_apps, :buyer_plan_change_permission, :buyer_can_select_plan ].map { |setting| friendly_service_setting(service, setting) }.compact.each do |setting|
             li.item
               = setting
-          - if provider_can_use?(:api_as_product)
             li.item
```

Comment:
This is not well aligned. Right not it is inside the `map` of setting above 😄 
It should be:
```ruby
          li.item
            ' ID for API calls is
            strong
              => service.id
            ' and system name is
            strong
              = service.system_name

          - [:buyers_manage_keys, :buyers_manage_apps, :buyer_plan_change_permission, :buyer_can_select_plan ].map { |setting| friendly_service_setting(service, setting) }.compact.each do |setting|
            li.item
              = setting
          li.item
            strong> Backends
            ' used in this product:
            ul
              - service.backend_apis.each do |backend_api|
                li = link_to backend_api.name, provider_admin_backend_api_path(backend_api)
```

Reply:
Good catch!